README.md

```diff
@@ -7,7 +7,7 @@
 [![Slack channel](https://img.shields.io/badge/chat-slack-green.svg?logo=slack)](https://kotlinlang.slack.com/messages/coroutines/)
```

Comment:
This file has been apparently changed on master but not on develop - with PR: Improve readability. #2563